computational-nuclear-activity.ipynb

```diff
@@ -116,6 +109,23 @@
     "    return activity"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Test your function by calculating the activity at $t = 500 $ seconds of a radionuclide with an initial concentration of $10^{24}$ atoms and a decay constant of $\\lambda = 10^{-3}$ seconds$^{-1}$."
```

Comment:
I'm now wondering if showing them how to calculate with half-life is too much? Do we want to add a half life here instead and then tell them what the decay constant is too if they want to check their work?

Reply:
Yeah good idea on the half life.